src/app/core/core.module.ts

```diff
@@ -1,10 +1,7 @@
-/* @format */
```

Comment:
Let's not get rid of the Prettier format pragma.

Reply:
I got rid of it because prettier was complaining for some reason, and by removing them it worked. Do you know why that happened @meisekimiu ?

Reply:
Can you show me the error? These lines are currently what enables Prettier to check the codebase in Github actions so if we remove them then we will not check the files.